docs/guides/mozilla-sops.md

```diff
@@ -213,9 +213,57 @@ IAM Role example:
 
 #### Azure
 
-When using Azure Key Vault you need to authenticate the kustomize controller either by passing
-[Service Principal credentials as environment variables](https://github.com/mozilla/sops#encrypting-using-azure-key-vault)
-or with [add-pod-identity](https://github.com/Azure/aad-pod-identity).
+When using Azure Key Vault you need to authenticate kustomize-controller either with [add-pod-identity](../use-cases/azure.md#aad-pod-identity)
+or by passing [Service Principal credentials as environment variables](https://github.com/mozilla/sops#encrypting-using-azure-key-vault).
+
+Create the Azure Key-Vault:
+
+```sh
+export VAULT_NAME="fluxcd-$(uuidgen | tr -d - | head -c 16)"
+export KEY_NAME="sops-cluster0"
+
+az keyvault create --name "${VAULT_NAME}"
+az keyvault key create --name "${KEY_NAME}" \
+  --vault-name "${VAULT_NAME}"
+  --protection software \
+  --ops encrypt decrypt
+az keyvault key show --name "${KEY_NAME}" \
+  --vault-name "${VAULT_NAME}" \
+  --query key.kid
+```
+
+If using AAD Pod-Identity, create an identity within Azure to bind against, then create an `AzureIdentity` object to match:
```

Comment:
Reference docs on how to create this identity and bind RBAC against it in Azure

Reply:
Are these YAML comments sufficient?
https://github.com/fluxcd/flux2/pull/1064/files#diff-3b71a8f9c2fc327cdc61557a81a129d04246171f15aa8b05668e2ff536211302R238-R243

Reply:
Yes, this looks good to me :)